packages/wdio-cli/src/interface.js

```diff
@@ -164,204 +180,37 @@ export default class WDIOCLInterface extends EventEmitter {
-            return logUpdate(this.display.join('\n') + '\n\n' + shutdownMessage)
-        }
-
-        if(isFinished || this.inDebugMode) {
-            return
-        }
-
-        /**
-         * check if environment supports ansi or does not
-         * support TTY and print a limited update if not
-         */
-        if (!this.hasAnsiSupport || !this.isTTY) {
-            /**
-             * only update if a job finishes
-             */
-            if (!wasJobCleared) {
-                return
-            }
-
-            const clockSpinnerSymbol = this.getClockSymbol()
-            return this.display.push([
-                `${clockSpinnerSymbol} ` +
-                `${this.jobs.size} running, ` +
-                `${this.result.passed} passed, ` +
-                (this.result.retries ? `${this.result.retries} retries, ` : '') +
-                `${this.result.failed} failed, ` +
-                `${totalJobs} total ` +
-                `(${Math.round((this.result.finished / totalJobs) * 100)}% completed)`].join(' '))
-        }
-
-        this.clearConsole()
-        this.display.push('')
-
-        /**
-         * print running jobs
-         */
-        for (const [cid, job] of Array.from(this.jobs.entries()).slice(0, MAX_RUNNING_JOBS_DISPLAY_COUNT)) {
-            const filename = job.specs.join(', ').replace(process.cwd(), '')
-            this.display.push([
-                chalk.bgYellow.black(' RUNNING '),
-                cid,
-                'in',
-                getRunnerName(job.caps),
-                '-',
-                filename
-            ].join(' '))
-        }
-
-        /**
-         * show number of pending and running jobs
-         */
-        if (pendingJobs || runningJobs > MAX_RUNNING_JOBS_DISPLAY_COUNT) {
-            const logString = []
-            if (runningJobs > MAX_RUNNING_JOBS_DISPLAY_COUNT) {
-                logString.push(
-                    runningJobs - MAX_RUNNING_JOBS_DISPLAY_COUNT,
-                    'running test suites' + (pendingJobs ? ' -' : ''))
-            }
-            if (pendingJobs) {
-                logString.push(pendingJobs, 'pending test suites')
-            }
-            this.display.push(chalk.yellow('...', ...logString.filter(l => Boolean(l))))
-        }
-
-        /**
-         * print reporters in watch mode
-         */
-        if (this.isWatchMode) {
-            this.printReporters()
-        }
-
-        /**
-         * add empty line between "pending tests" and results
-         */
-        if (this.jobs.size) {
-            this.display.push('')
-        }
-
-        this.printStdout(10)
-        this.printSummary()
-        this.updateClock()
+        const isFinished = this.jobs.size === 0
+        const shutdownMessage = !isFinished ? 'Ending WebDriver sessions gracefully ...\n' + '(press ctrl+c again to hard kill the runner)' : 'Ended WebDriver sessions gracefully after a SIGINT signal was received!'
```

Comment:
It might be better use `isRunning` instead of `!isFinished`

Reply:
I'll check, I have copied most of the logic, like these two lines.

Reply:
changed to isRunning